cmd/kubeadm/app/apis/kubeadm/v1alpha3/defaults.go

```diff
@@ -42,9 +42,6 @@ const (
 	DefaultImageRepository = "k8s.gcr.io"
 	// DefaultManifestsDir defines default manifests directory
 	DefaultManifestsDir = "/etc/kubernetes/manifests"
-	// DefaultClusterName defines the default cluster name
```

Comment:
I don't think we should move this cost alone because we end in a mixed situation where some API defaults are defined here on some in another place

Reply:
I'm restoring this and using `v1beta1` package instead for now.